### PR TITLE
Add bump.sh reference in the Documentation category

### PR DESCRIPTION
**Pull request by scharrier**
Disclaimer — I am the founder of [bump.sh](https://bump.sh).

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -206,6 +206,14 @@
   v2: true
   v3: true
 
+- name: Bump
+  category: documentation
+  link: https://bump.sh
+  language: SaaS
+  description: Bump generates elegant documentations and automatic changelogs from your Swagger/OpenAPI specifications. Git diff, for your API.
```
Comment (philsturgeon) [inline _data/tools.yml]: Can we remove Swagger from this description? They changed their name a long time ago, and at some point we need to just accept that its not the name anymore.

Comment (philsturgeon) [inline _data/tools.yml]: ### 
```suggestion
  description: Bump generates elegant documentations, and automatically generates changelogs from your OpenAPI specifications. Git diff, for your API.
```

Comment (scharrier) [reply _data/tools.yml]: Yes, you're totally right. Change made.

**Comment by philsturgeon** [comment]
Thank you!

**Pull request merged**